### PR TITLE
Feature: add gas price estimation feature to zcnbridge

### LINKED ISSUE #1426: Feature: add gas price estimation feature to zcnbridge

### PR DESCRIPTION
**Pull request by YarikRevich**

### CODE REVIEW

File: zcnbridge/bridge_test.go [hunk 1]
```diff
@@ -39,7 +40,12 @@ import (
 
 const (
 	ethereumAddress = "0xD8c9156e782C68EE671C09b6b92de76C97948432"
-	password        = "02289b9"
+
+	alchemyEthereumNodeURL = "https://eth-mainnet.g.alchemy.com/v2/9VanLUbRE0pLmDHwCHGJlhs9GHosrfD9"
+	infuraEthereumNodeURL  = "https://mainnet.infura.io/v3/7238211010344719ad14a89db874158c"
+	value                  = 1e+10
+
+	password = "02289b9"
```
Comment (peterlimg) [inline zcnbridge/bridge_test.go:48]: Let's put the info above into github ENVs or SECRETS

Comment (YarikRevich) [reply zcnbridge/bridge_test.go:48]: The given data is fake, there is not security issue.

Comment (YarikRevich) [reply zcnbridge/bridge_test.go:48]: It's used only for tests

**Review by peterlimg — APPROVED** [review]
lgtm

**Pull request merged**